docs/usage/merge-confidence.md

```diff
@@ -25,11 +25,14 @@ Merge Confidence badges for pull requests are available on any supported platfor
 
 ## Supported languages
 
-Data is available for packages from:
-
-- npm
-- Maven
-- PyPI
+Data is available for the following languages (from the noted data source):
+
+- JavaScript (npm)
+- Java (maven)
+- Python (pypi)
+- .NET (nuget)
+- PHP (packagist)
+- Ruby (rubygems)
```

Comment:
```suggestion
| Language   | Datasource  |
| ---------- | ----------- |
| JavaScript | `npm`       |
| Java       | `maven`     |
| Python     | `pypi`      |
| .NET       | `nuget`     |
| PHP        | `packagist` |
| Ruby       | `rubygems`  |
```

Let's use a table instead of a list with ugly `()`. 😉